metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/ShellFunctionsTest.java

```diff
@@ -40,8 +45,8 @@
   );
 
   Context context = new Context.Builder()
-            .with(Context.Capabilities.SHELL_VARIABLES , () -> variables)
-            .build();
+            .with(Context.Capabilities.SHELL_VARIABLES , () -> variables).build();
```

Comment:
I am actually interested in what direction as a project we should be taking with these types of fluent, chained statements.  I run across this all the time and I want to know the 'right' way that I should be doing it for the project.

IMHO, the way it was (separated by a line break) is more readable.  Meaning, a long set of chained statements should be separated by line breaks.  For example...
```
      result = new ProfileMeasurement()
              .withProfileName(profileName)
              .withEntity(entity)
              .withGroups(groups)
              .withPeriod(period)
              .withProfileValue(profileValue)
              .withTriageValues(triageValues)
              .withDefinition(definition);
```

But, of course, in terms of code style my opinion doesn't matter.  It is all about our style guidelines. 
 What does the Google code style guidelines say?   

Doesn't [this](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break) support what I have said above about line breaks in this case?

Reply:
I didn't reformat that like that on purpose.  I had a period where I was setting the CONSOLE capability.  When I removed it, it just worked out like this.

Even if I select and format in intellij it doesn't change the .build() to the next line.

I think what you have above is fine.  I would like it to 'just' happen if I format code though, since it is easy for things to slip through.

It is tough right now, because so much of the codebase isn't formatted to check style, and I don't think we want every pr to include a lot of formatting changes.

Reply:
How do you have your formatting preferences set to get the above?

Reply:
Right now, I never let my IDE reformat for me.  Like you said, if we get the code base matching check style and I can load that style into my IDE, then I'd gladly let it do most of the work for me.

Maybe I'll open a discuss thread.  I don't know how to handle this kind of thing and it happens all the time.

But for this specific scenario in your PR, it really doesn't matter either way.  I think you're good to go either way.